src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/helper/StorageHelper.java

```diff
@@ -26,6 +30,10 @@ private StorageHelper() {
 
     public static void uploadZipFile(Container container, ZipArchive zipArchive) {
         String sasToken = SasTokenRetriever.getTokenFor(container.name);
+        BlobRequestConditions blobRequestConditions = new BlobRequestConditions();
+        blobRequestConditions.setIfNoneMatch(Constants.HeaderConstants.ETAG_WILDCARD);
```

Comment:
what is the significance of `ETAG_WILDCARD` ?

Reply:
just to ignore if it is there,  means override true.